pkg/object/meshcontroller/master/master.go

```diff
@@ -79,7 +79,7 @@ func New(superSpec *supervisor.Spec) *Master {
 	}
 	m.maxHeartbeatTimeout = heartbeat * 2
 
-	go m.run()
+	m.run()
```

Comment:
Why block here, would not it return master？

Reply:
The implementation of `run` is asynchronous. So it won't be blocked here.

Reply:
Then this code isn't basing on the latest code..

Reply:
Then this code isn't basing on the latest code..

Reply:
Yes, I found that too. It's not based on the latest code. Let me fix it. Sorry

Reply:
Fixed. @xxx7xxxx